examples/l5pc/hocmodel.py

```diff
@@ -37,6 +37,8 @@ def instantiate(self):
         """Instantiate model in simulator"""
 
         self.cell = bglibpy.Cell(self.template, self.morphology)
+
+        self.cell.re_init_rng()
```

Comment:
Would it be possible to use the bglibpy Simulation() object here ? It would wrap this kind of setup problems.